InAppMessaging/Example/Tests/FIRIAMMessageContentDataWithImageURLTests.m

```diff
@@ -147,32 +206,34 @@ - (void)testReportErrorOnGeneralNSErrorFromNSURLSession {
   OCMVerify([self.mockedNSURLSession dataTaskWithRequest:[OCMArg any]
                                        completionHandler:[OCMArg any]]);
 
-  // by this time we should have capturedCompletionHandler being the callback block for the
-  // NSURLSessionDataTask, now feed it with an NSError see how the block from
-  // loadImageDataWithBlock: would react to it.
-  capturedCompletionHandler(nil, nil, customError);
+  for (void (^capturedCompletionHandler)(NSData *data, NSURLResponse *response, NSError *error)
+           in completionHandlers) {
+    capturedCompletionHandler(nil, nil, customError);
+  }
 
   [self waitForExpectationsWithTimeout:5.0 handler:nil];
 }
 
 - (void)testReportErrorOnNonImageContentTypeResponse {
-  __block void (^capturedCompletionHandler)(NSData *data, NSURLResponse *response, NSError *error);
+  // Used to capture both portrait and landscape callbacks.
+  NSMutableArray *completionHandlers = [NSMutableArray array];
 
   OCMStub([self.mockedNSURLSession
       dataTaskWithRequest:[OCMArg any]
-        completionHandler:[OCMArg checkWithBlock:^BOOL(id completionHandler) {
-          capturedCompletionHandler = completionHandler;
+        completionHandler:[OCMArg checkWithBlock:^BOOL(void (^capturedCompletionHandler)(
+                              NSData *data, NSURLResponse *response, NSError *error)) {
+          [completionHandlers addObject:capturedCompletionHandler];
           return YES;
-        }]  // second parameter is the callback which we don't care in this unit testing
-  ]);
+        }]]);
 
   XCTestExpectation *expectation =
       [self expectationWithDescription:@"image load callback triggered."];
   [_defaultContentDataWithImageURL
-      loadImageDataWithBlock:^(NSData *_Nullable imageData, NSError *_Nullable error) {
+      loadImageDataWithBlock:^(NSData *_Nullable imageData, NSData *_Nullable landscapeImageData,
+                               NSError *error) {
         XCTAssertNil(imageData);
         XCTAssertNotNil(error);  // we should report error due to the http response
-                                 // content type being invalid
+        // content type being invalid
```

Comment:
Can you move this whole comment above the `XCTAssertNotNil`? Right now it's a bit confusing. Also, `s/http/HTTP/` , capitalize `We`, and add punctuation.